Cargo.toml

```diff
@@ -17,7 +17,7 @@ authors = [
 [features]
 
 [dependencies]
-async-std = { version = "1.6.0", features = ["unstable"] }
+futures-lite = "1.11.3"
 http-types = "2.0.1"
```

Comment:
`http-types` also pulls in async-std. Could you change this to this so we aren't required to pull in all of its optional dependencies?

```
http-types = { version = "2.0.1", default-features = false }
```